src/embind/embind.js

```diff
@@ -915,27 +915,31 @@ var LibraryEmbind = {
     var returns = (argTypes[0].name !== "void");
 
 #if DYNAMIC_EXECUTION == 0
-    var argsWired = new Array(argCount - 2);
+    var expectedArgCount = argCount - 2;
+    var argsWired = new Array(expectedArgCount);
+    var invokerFuncArgs = [];
+    var destructors = [];
     return function() {
-      if (arguments.length !== argCount - 2) {
-        throwBindingError('function ' + humanName + ' called with ' + arguments.length + ' arguments, expected ' + (argCount - 2) + ' args!');
+      if (arguments.length !== expectedArgCount) {
+        throwBindingError(`function ${humanName} called with ${arguments.length} arguments, expected ${expectedArgCount} args!`);
```

Comment:
Template literals are an ES6 feature. @kripken do you know how much we care about that? As far as I know we've been generally avoiding ES6 features in common code, though maybe it makes sense to start assuming ES6 if we have Wasm.

Reply:
Went back to 5.